<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTools</class>
 <widget class="QDialog" name="QTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_search">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
